app/src/main/java/org/simple/clinic/medicalhistory/newentry/NewMedicalHistoryScreen.kt

```diff
@@ -172,6 +174,24 @@ class NewMedicalHistoryScreen : BaseScreen<
     }
   }
 
+  override fun onAttach(context: Context) {
+    super.onAttach(context)
+    requireContext().injector<Injector>().inject(this)
```

Comment:
Just an FYI, we can use the `context` from the `onAttach` directly. There is no need to use `requireContext` here. It doesn't make any difference, just a visual consistency with other screens in the project.

Reply:
Oh right, my bad.